src/main/kotlin/com/chattriggers/ctjs/minecraft/objects/display/Display.kt

```diff
@@ -115,6 +184,21 @@ abstract class Display() {
         } else lines.add(index, toAdd)
     }
 
+
+    /**
+     * Removes a line from the display
+     *
+     * @param line The index of the line, a string or the DisplayLine to remove
+     */
+    fun removeLine(line: Any) = apply {
+        when (line) {
+            is Int -> lines.removeAt(line)
+            is DisplayLine -> lines.remove(line)
+            is String -> lines.forEach { if (it.getText() == line) lines.remove(it) }
```

Comment:
```suggestion
            is String -> lines.removeIf { it.getText() == line }
```

Reply:
Actually, that won't do it since `getText` returns a `Text`, not a string, but still.